docs/src/qsa-cli/installation.md

```diff
@@ -4,6 +4,7 @@ From sources:
 
 ```` console
 $ cd qsa-cli
+$ virtualenv --system-site-packages venv  # system aware for pyqgis
```

Comment:
`pyqgis` is not used by `qsa-cli`, so `--system-site-packages` is not necessary

Reply:
alright, i just tried it and it worked as well
( originally, I just copied the command from the other sub-projects)